Homework2/Homework2/Project.cs

```diff
@@ -0,0 +1,73 @@
+namespace Homework2;
+
+public class Project
+{
+    private string _name;
+    private string _description;
+    private string _deadline;
+    private decimal _budget;
+    private List<Staff> _staff;
+    private List<Tasks> _tasks;
```

Comment:
To prevent bugs it's better to provide default value for lists, e.g `private List<Staff> _staff = new List<Staff>();`

Reply:
Fixed